vivo-project/Vitro
# Upgrade of slf4j-log4j12 dependency

**[VIVO GitHub issue](https://github.com/vivo-project/VIVO/issues)**: [3913](https://github.com/vivo-project/VIVO/issues/3913)

# What does this pull request do?
Resolve security vulnerabilities with log4j 1.2.17 

# What's new?
Upgrade slf4j-log4j12 version in [pom.xml file](https://github.com/vivo-project/Vitro/blob/main/dependencies/pom.xml#L215)

# How should this be tested?
* mvn clean install
* check whether tests passing
* run vivo, turn on developer panel and logging in, check whether there is any issue with logging

# Interested parties
Tag (@ mention) interested parties or, if unsure, @VIVO-project/vivo-committers
